### DISCUSSION TITLE
Zeroing spindle degrees

### DISCUSSION THREAD

**Discussion by tektonandroid** [kachurovskiy/nanoels#180] [Q&A] [answered]
Hello all. It is posible to zeroing spindle spin for easy dividing operations with H4?
Thanks

**Comment by tektonandroid** [comment, marked as answer]
I just discovered how to do it.  Only by pressing STOP.
Regards

**Reply by kachurovskiy** [reply]
Yes, sorry for not replying sooner ✨
